packages/button/src/component.tsx

```diff
@@ -28,25 +28,80 @@ export const Button = forwardRef<
     ...rest
   } = props;
 
+   const ccButton = {
```

Comment:
This one is checked in ONLY for code reviewing and testing purposes. This must be deleted from here before merge and moved to css repo.